pages/api/profiles/[username]/index.js

```diff
@@ -51,6 +51,16 @@ export async function getUserApi(req, res, username, options = {}) {
     {
       $match: { username },
     },
+    {
+      $set:{
+        "milestones":{
+          $sortArray:{
+            input:"$milestones",
+            sortBy: {date:1}
```

Comment:
```suggestion
            sortBy: {date:-1}
```
As discussed on the issue it should show the most recent ones.
![image](https://github.com/EddieHubCommunity/BioDrop/assets/93700622/d039b359-ade0-4fa9-af01-305a6013b23f)